test/videojs-hls_test.js

```diff
@@ -1720,6 +1739,11 @@ test('calling play() at the end of a video resets the media index', function() {
   player.hls.ended = function() {
     return true;
   };
+
+  player.hasClass = function(classname) {
```

Comment:
hasClass isn't defined in this scope, but you recommended using it in a previous PR review, so this addition keeps the test working.